src/main/java/com/zenjava/javafx/maven/plugin/JarMojo.java

```diff
@@ -249,22 +277,54 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                 File dest = new File(libDir, artifactFile.getName());
                 if( !dest.exists() ){
                     try{
-                        Files.copy(artifactFile.toPath(), dest.toPath());
+                        if( !skipCopyingDependencies ){
+                            Files.copy(artifactFile.toPath(), dest.toPath());
+                        } else {
+                            getLog().info(String.format("Skipped copying classpath element: %s", artifactFile.getAbsolutePath()));
+                        }
                     } catch(IOException ex){
                         getLog().warn(String.format("Couldn't read from file %s", artifactFile.getAbsolutePath()));
                         getLog().debug(ex);
                         brokenArtifacts.add(artifactFile.getAbsolutePath());
                     }
                 }
-                classpath.append("lib/").append(artifactFile.getName()).append(" ");
+                classpath.append(libFolderName).append("/").append(artifactFile.getName()).append(" ");
```

Comment:
I would recommend to introduce constants, e.g.:
private static final String MANIFEST_CLASSPATH_FILE_SEPARATOR = "/";
private static final String MANIFEST_CLASSPATH_PATH_SEPARATOR = " ";